doc/arch/010-user-permissions.md

```diff
@@ -1,4 +1,8 @@
 # User Permissions
+## This permission model is deprecated (October 2020)
```

Comment:
Let's not do this in the ADRs. ADRs are meant to be immutable.

Reply:
Hmm, how can we indicate this is no longer relevant though? Don't want to mislead someone looking at these in the future.

Reply:
We could [write an ADR next week](https://app.clubhouse.io/simpledotorg/story/1227/write-an-adr-to-document-how-the-new-permissions-system-works) 😄

Reply:
Would still want to say this, and link to that one instead of the PRD...

Reply:
Looks like an ADR's `status` can be updated: using `deprecated` or `superseded` (with a link)
Will do that instead.

Examples:
https://peterevans.dev/posts/lightweight-architecture-decision-records/
https://backstage.io/docs/architecture-decisions/adrs-overview
https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions

Reply:
Perfect 👍